goinsta.go

```diff
@@ -1097,8 +1097,17 @@ func (insta *Instagram) TotalUserFollowers(userID int64) (response.UsersResponse
 	}
 }
 
-func (insta *Instagram) GetRecentActivity() ([]byte, error) {
-	return insta.sendSimpleRequest("news/inbox/?")
+func (insta *Instagram) GetRecentActivity() (response.RecentActivityResponse, error) {
+	result := response.RecentActivityResponse{}
+	bytes, err := insta.sendSimpleRequest("news/inbox/?")
```

Comment:
Good , But now we have it on [Updated function of GetRecentActivity](https://github.com/ahmdrz/goinsta/blob/82b8eb0d9a3b687b2be5809fc0946a68d5170875/goinsta.go#L1101)